windows_containers/creating_windows_machinesets/creating-windows-machineset-nutanix.adoc

```diff
@@ -0,0 +1,19 @@
+:_content-type: ASSEMBLY
+[id="creating-windows-machineset-nutanix"]
+= Creating a Windows MachineSet object on Nutanix
+include::_attributes/common-attributes.adoc[]
+:context: creating-windows-machineset-nutanix
+
+toc::[]
+
+You can create a Windows `MachineSet` object to serve a specific purpose in your {product-title} cluster on Nutanix. For example, you might create infrastructure Windows machine sets and related machines so that you can move supporting Windows workloads to the new Windows machines.
+
```

Comment:
Just checking if this assembly needs a "Prerequisites" section like the other cloud providers do?

Reply:
I was told the prerequisites were not needed and asked to remove them during QE review. Will verify.

Reply:
Woops! Never mind. I misunderstand what the reviewer was asking me to remove. Thank you!